src/io/flutter/sdk/FlutterSdkUtil.java

```diff
@@ -49,9 +49,9 @@ private FlutterSdkUtil() {
   /**
    * Return the environment variable value to use when shelling out to the Flutter command-line tool.
    */
-  public static String getFlutterEnvValue() {
+  public static String getFlutterHostEnvValue() {
     final String clientId = ApplicationNamesInfo.getInstance().getFullProductName().replaceAll(" ", "-");
-    final String existingVar = System.getenv(FLUTTER_ENVIRONMENT);
+    final String existingVar = System.getenv(FLUTTER_HOST_ENV);
     return existingVar == null ? clientId : (existingVar + ":" + clientId);
```

Comment:
Okay, fair enough. Maybe this is over-engineering, but I was thinking we could either send a set of hashtags as JSON:
```
["travis","intellij-idea"]
```
Or a map as JSON:
```
{"travis":true,"product":"IntelliJ IDEA"}
```
And let the JSON encoder/parser worry about special characters. (Reasonably compact, and better than getting into an ad-hoc user-agent parsing situation later.)